jerry-core/lit/lit-strings.cpp

```diff
@@ -757,6 +757,15 @@ lit_utf8_string_code_unit_at (const lit_utf8_byte_t *utf8_buf_p, /**< utf-8 stri
   return code_unit;
 } /* lit_utf8_string_code_unit_at */
 
+/* CESU-8 number of bytes occupied lookup table */
+const lit_utf8_size_t table[]
```

Comment:
Seems OK, but I would reduce the table size to uint8 from uint32 to reduce the table size from 64 to 16 bytes (the values are small anyway). Also the table should be aligned to 16 bytes.